README.md

```diff
@@ -0,0 +1,33 @@
+# CASS-Degrees-Code
```

Comment:
Title doesn't need to be dashed :P

Reply:
Haha the title was actually auto generated by GitHub. Didn't even notice it :')